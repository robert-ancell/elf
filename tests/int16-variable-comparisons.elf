int16 value1 = 1
int16 value2 = 1
int16 value3 = 2

print (value1 == value2)
print (value1 == value3)

print (value1 != value2)
print (value1 != value3)

print (value1 > value2)
print (value1 >= value2)
print (value1 < value2)
print (value1 <= value2)

print (value1 > value3)
print (value1 >= value3)
print (value1 < value3)
print (value1 <= value3)

print (value3 > value1)
print (value3 >= value1)
print (value3 < value1)
print (value3 <= value1)
