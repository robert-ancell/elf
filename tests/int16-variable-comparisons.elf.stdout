true
false
false
true
false
true
false
true
false
false
true
true
true
true
false
false
